testing/scenarios/auto_detect_config/main.go

```diff
@@ -1,3 +1,9 @@
 package main
 
 // TODO - malformed todo
+
+// TODO #1: closed issue
+
+// TODO 2: closed issue
+
+// TODO #9999999: non-existent issue
```

Comment:
Please create a new scenario where you're testing your new feature in isolation.